media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format.  flac123 is the only \
```

Comment:
This is standard english typography: https://en.wikipedia.org/wiki/Sentence_spacing
But yes, many people (even english) consider it strange and we usually don't use it.